dev-proxy/ProxyHost.cs

```diff
@@ -346,9 +346,53 @@ public RootCommand GetRootCommand(ILogger logger)
         var outdatedShortOption = new Option<bool>("--short", "Return version only");
         outdatedCommand.AddOption(outdatedShortOption);
         outdatedCommand.SetHandler(async versionOnly => await OutdatedCommandHandler.CheckVersionAsync(versionOnly, logger), outdatedShortOption);
-
         command.Add(outdatedCommand);
 
+        var jwtCommand = new Command("jwt", "Manage JSON Web Tokens ");
+        var jwtCreateCommand = new Command("create", "Create a new JWT token");
+        var jwtSchemeOption = new Option<string>("--scheme", "The scheme name to use for the token.");
+        jwtSchemeOption.SetDefaultValue("Bearer");
+        jwtCreateCommand.AddOption(jwtSchemeOption);
+        var jwtNameOption = new Option<string>("--name", "The name of the user to create the token for.");
+        jwtNameOption.AddAlias("-n");
+        jwtNameOption.SetDefaultValue("Dev Proxy");
+        jwtCreateCommand.AddOption(jwtNameOption);
+        var jwtAudienceOption = new Option<string>("--audience", "The audiences to create the token for.");
+        jwtAudienceOption.AddAlias("-a");
+        jwtAudienceOption.SetDefaultValue("https://myserver.com");
+        jwtCreateCommand.AddOption(jwtAudienceOption);
+        var jwtIssuerOption = new Option<string>("--issuer", "The issuer of the token.");
+        jwtIssuerOption.AddAlias("-i");
+        jwtIssuerOption.SetDefaultValue("dev-proxy");
+        jwtCreateCommand.AddOption(jwtIssuerOption);
+        var jwtRolesOption = new Option<string>("--roles", "A role claim to add to the token. Specify once for each scope. Use a comma to separate scopes.");
```

Comment:
Instead of using comma-separated values, let's use the native support for multiple values. That way we don't need parse the string manually and there's less room for error